BasicAlgorithms/BubbleSort.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections;
+
+namespace BasicAlgorithms
+{
+    public class BubbleSort
+    {
+        public int[] ArrayOfIntegers { get; set; }
```

Comment:
This name is not suggestive. Please use a name which refers exactly to your business logic (e.g numbers, items, etc)